### PR TITLE
Update Swift entries to link to the monorepo

### PR DESCRIPTION
**Pull request by grdsdev**
## What kind of change does this PR introduce?

Docs update

## What is the current behavior?

Swift currently links to individual repositories

## What is the new behavior?

Swift libraries are now a monorepo, so I'm updating to links to the monorepo.

### CODE REVIEW

**Review by dshukertjr — APPROVED** [review]
Sweat! Thanks for the update!

**Comment by grdsdev** [comment]
@dshukertjr, thank you. Feel free to merge it once you can, it doesn't seem I have permission to, even after your approval.

**Pull request merged**
